octodns/provider/ns1.py

```diff
@@ -35,11 +37,38 @@ def __init__(self, id, api_key, *args, **kwargs):
         self._client = NSONE(apiKey=api_key)
 
     def _data_for_A(self, _type, record):
-        return {
+        # record meta (which would include geo information is only
+        # returned when getting a record's detail, not from zone detail
+        geo = {}
+        data = {
             'ttl': record['ttl'],
             'type': _type,
-            'values': record['short_answers'],
         }
+        values, codes = [], []
+        if 'answers' not in record:
```

Comment:
Most of this section of code doesn't have code coverage in the tests. `./script/coverage` will run the tests with coverage and drop the results in `./htmlcov`.

Reply:
added test mocks to get coverage back up.